DeployGovCloudJenkinsfile

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env groovy
+def devcloudArtServer = Artifactory.server('devcloud')
+pipeline {
+    agent none
+    options {
+        skipDefaultCheckout()
+        buildDiscarder(logRotator(artifactDaysToKeepStr: '1', artifactNumToKeepStr: '1', daysToKeepStr: '5', numToKeepStr: '10'))
+    }
+    parameters  {
+        choice(name: 'DEPLOYMENT_TYPE', choices:'govw', description: 'This specifies which point of presence to deploy to')
+        booleanParam(name: 'MAP_ROUTES_TO_DEPLOYED_APP', defaultValue: true, description: 'Determines if the published routes are mapped to the newly deployed app. Defaults to true.')
+        booleanParam(name: 'RUN_FULL_ACCEPTANCE_TESTS', defaultValue: true, description: 'Determines whether to run the full acceptance test suite against the freshly deployed UAA. If left unchecked, it will run at least the basic acceptance tests.')
+    }
+    stages {
+		stage('Deploy to govcloud') {
+			environment {
+                CF_CREDENTIALS = credentials("CF_CREDENTIALS_${DEPLOYMENT_TYPE.toUpperCase().replaceAll('-','_')}")
+                ADMIN_CLIENT_SECRET = credentials("ADMIN_CLIENT_SECRET_${DEPLOYMENT_TYPE.toUpperCase().replaceAll('-','_')}")
+            }
+            agent {
+                docker {
+                    image 'dig-propel/govcloud-deploy'
+                    label 'dind'
+                    args "--privileged --sysctl net.ipv6.conf.all.disable_ipv6=0"
+                    registryUrl 'https://registry.gear.ge.com/'
+                    registryCredentialsId "DTR_CREDS"
+                }
+            }
+
+			steps {
+				script {
+                    dir('uaa') {
+                        checkout scm
+                    }
+                    APP_VERSION = sh (returnStdout: true, script: '''
+                        grep 'version' uaa/gradle.properties | sed 's/version=//'
+                        ''').trim()
+                    APP_VERSION = '4.6.4'
```

Comment:
this is on purpose since we're not 100% on 4.10.0 in govcloud yet. since this will only be available on 4.10.0 branches, it will try to use that `APP_VERSION` so it needs to be hardcoded to 4.6.4 temporarily